modules/docs/src/Volo.Docs.Domain/Volo.Docs.Domain.csproj

```diff
@@ -20,14 +20,20 @@
     <PackageReference Include="Microsoft.Extensions.Http" />
     <PackageReference Include="Octokit" />
     <PackageReference Include="NEST" />
-    <PackageReference Include="Microsoft.Extensions.FileProviders.Embedded" />
+    <PackageReference Include="Microsoft.Extensions.FileProviders.Embedded" />       
+    <PackageReference Include="itext.bouncy-castle-adapter" />
+    <PackageReference Include="itext7" />
+    <PackageReference Include="itext7.pdfhtml" />
+    <PackageReference Include="Markdig.Signed" />
+    <PackageReference Include="Scriban" />
```

Comment:
```suggestion
```

Already exists in modules/docs/src/Volo.Docs.Domain.Shared/Volo.Docs.Domain.Shared.csproj. No need to add it again here.